app/components/feedback_component.html.erb

```diff
@@ -4,6 +4,20 @@
     <ul class="list-unstyled">
       <li class="feedback-link">
         <%= link_to t("feedback.report_error"), helpers.error_feedback_url(@id) %>
+        <a href="https://reftracker.nla.gov.au/reft100.aspx?key=Cultural_Feedback&qnudftb17=https://catalogue.nla.gov.au/catalog/1698874">
```

Comment:
Needs the specific page removed from the url (same as for the environment vars above) and insert the current page